test/jdk/javax/net/ssl/SSLSocket/Tls13PacketSize.java

```diff
@@ -72,6 +72,7 @@ protected void runServerApplication(SSLSocket socket) throws Exception {
 
         sslOS.write(appData);
         sslOS.flush();
+        sslIS.read(appData, 1, appData.length-1);
```

Comment:
```suggestion
        int drained = 1;
        while (drained < appData.length) {
            drained += sslIS.read(appData, drained, appData.length - drained);
        }
```
this one actually works, I checked.